src/utils/DeviceConnectionChecker.cpp

```diff
@@ -0,0 +1,46 @@
+#include "DeviceConnectionChecker.h"
+#include "AppGui.h"
+
+const QString DeviceConnectionChecker::CONNECTED_DEVICES_GROUP = "ConnectedDevices/";
```

Comment:
out of curiosity, we didn't have a similar service before? for example, in our logic to check the change number for a given user

Reply:
We have DbExportsRegistry: https://github.com/mooltipass/moolticute/blob/master/src/DbExportsRegistry.cpp
It is embedded in DbBackup codes, so was not able to reuse it, it writes dbExportsRegistry.ini with the changeNumbers for given cardId.
I would be able to use that file, but I believe a new connectedDevices.ini much more clean.